examples/cookbook/agents/generating_sql_agent.py

```diff
@@ -97,7 +98,7 @@ def _get_response(self, question: str = ""):
         ]
         return self._get_response("")
```

Comment:
nit: remove return here as it's unnecessary with return type `None` (see above comment)